frontend/components/card/CardTextEntry.vue

```diff
@@ -20,7 +20,7 @@
         <!-- Desktop -->
         <textarea
           id="feedback"
-          class="hidden w-full px-3 py-2 overflow-y-scroll bg-transparent border border-white resize-none rounded-md md:block dark:border-white md:resize-y md:overflow-hidden"
+          class="hidden w-full resize-none overflow-y-scroll rounded-md border border-white bg-transparent px-3 py-2 md:block md:resize-y md:overflow-hidden dark:border-white"
```

Comment:
Just a note to myself that we have `border-white` for each here and that this should be a custom color of some kind. I'll get to this with the text component work.